### ISSUE TITLE
🐞 npm packages broken since 0.13.2

### ISSUE THREAD

**Issue by tminich** [dagger/dagger#8559]
### What is the issue?

The npm packages version 0.13.2 and .3 lack the dist folder. As I couldn't find any information about a change like that I doubt this was intentional?

### Dagger version

dagger v0.13.3 (registry.dagger.io/engine:v0.13.3) linux/amd64

**Comment by jedevc** [comment]
cc @TomChv

**Comment by TomChv** [comment]
Hey, yes it is intentional since #8494, we use `.ts` import extension instead of `.js`.

Also, the `dist/` folder isn't necessary to execute TypeScript module so you shouldn't need it.
Let me know if you have a specific need.

**Comment by TomChv** [comment]
Are you using Dagger from a module or you import it as an external library?

**Comment by tminich** [comment]
I'm not sure what's more applicable. The project is quite old by now, so I'm not yet using the new dagger command syntax (aka no `dagger init`, just running my ts code with `dagger run -d node --loader ts-node/esm`)

**Comment by TomChv** [comment]
> I'm not sure what's more applicable. The project is quite old by now, so I'm not yet using the new dagger command syntax (aka no dagger init, just running my ts code with dagger run -d node --loader ts-node/esm)

Ohhh because it should work with the new dagger module system! However I'm gonna revert the PR that broke your npm package because it's not worth it I think.
You should be fine for the next release :)

**Comment by TomChv** [comment]
@tminich Should we close that one since `tsx` & #8494 has been merge so it will be solve in the next release? (two weeks)

**Comment by tminich** [comment]
I'll take a look tomorrow. I've reverted to 0.12.7 for now as I ran into a different issue while using 0.13.1 and I didn't have much time for experimentation today =)

**Comment by tminich** [comment]
Doesn't work with tsx
```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/tminich/devel/pits-ci/node_modules/@dagger.io/dagger/dist/index.js' imported from /home/tminich/devel/pits-ci/lib/build.mts
```

But personally I'm good till the next release, I'll just stay on 0.12 till then

**Comment by moljac024** [comment]
Why was this closed? The npm package is still broken as of 0.13.3.

The npm package includes only 3 files:

- LICENCE
- README.md
- package.json (which references dist/index.js as the main entrypoint)

Installing it as dependency and trying to import anything from it will fail. If the intention is to not have users install the npm package any more but go through the dagger init system and generate the sdk code then this npm package needs to be deleted. 

As it is right now the npm package is actually useless. It is an empty, broken package that references non-existing files in its manifest.

**Comment by jedevc** [comment]
> Why was this closed? The npm package is still broken as of 0.13.3.

https://github.com/dagger/dagger/issues/8559#issuecomment-2373695987:

> https://github.com/dagger/dagger/pull/8494 has been merge so it will be solve in the next release? (two weeks)

The issue is fixed in main, the next release of dagger next week will contain the fix.
